tsl/test/sql/cagg_joins.sql.in

```diff
@@ -247,163 +386,196 @@ GROUP BY name, bucket, devices.device_id;
+CREATE MATERIALIZED VIEW cagg_unequal
 WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
 SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
    name
 FROM conditions, devices
 WHERE conditions.device_id <> devices.device_id
 GROUP BY name, bucket;
 
 --Unsupported join condition
-CREATE MATERIALIZED VIEW conditions_summary_daily_unequal
+CREATE MATERIALIZED VIEW cagg_unequal
 WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
 SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
    name
 FROM conditions, devices
 WHERE conditions.device_id = devices.device_id AND
       conditions.city like '%cow*'
 GROUP BY name, bucket;
 
 --Unsupported join condition
-CREATE MATERIALIZED VIEW conditions_summary_daily_unequal
+CREATE MATERIALIZED VIEW cagg_unequal
 WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
 SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
    name
 FROM conditions, devices
 WHERE conditions.device_id = devices.device_id OR
       conditions.city like '%cow*'
 GROUP BY name, bucket;
 
 --Error out when join type is not inner
-CREATE MATERIALIZED VIEW conditions_summary_daily_outer
+CREATE MATERIALIZED VIEW cagg_outer
 WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
 SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
    name
 FROM conditions FULL JOIN devices
 ON conditions.device_id = devices.device_id
 GROUP BY name, bucket;
 
-CREATE MATERIALIZED VIEW conditions_summary_daily_cagg
+CREATE MATERIALIZED VIEW cagg_outer
+WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   name
+FROM conditions LEFT JOIN devices
+ON conditions.device_id = devices.device_id
+GROUP BY name, bucket;
+
+CREATE MATERIALIZED VIEW cagg_cagg
 WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
 SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
    devices.device_id device_id,
    name
 FROM conditions, devices
 WHERE conditions.device_id = devices.device_id
 GROUP BY name, bucket, devices.device_id;
 
 --Join between cagg and normal table
-CREATE MATERIALIZED VIEW conditions_summary_daily_nested
+CREATE MATERIALIZED VIEW cagg_nested
 WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE) AS
 SELECT time_bucket(INTERVAL '1 day', cagg.bucket) AS bucket,
    devices.name
-FROM conditions_summary_daily_cagg cagg, devices
+FROM cagg_cagg cagg, devices
 WHERE cagg.device_id = devices.device_id
 GROUP BY 1,2;
 
+DROP MATERIALIZED VIEW cagg_nested CASCADE;
+
 --Error out for join between cagg and hypertable
```

Comment:
nit, might be better to place this test case further up with the rest of the error cases? (because there's the successful creation of cagg_nested in between now)